app/api/src/controllers/taskController.ts

```diff
@@ -112,3 +112,173 @@ export const updateTaskHandler = asyncHandler(
     });
   },
 );
+
+// @desc Creates new planned task list for next day
+// @route POST /task/plan
+// @access Private
+export const planTaskHandler = async (
+  req: AuthRequest,
+  res: Response,
+): Promise<void> => {
```

Comment:
Promise<void> isn't needed here. You should probably wrap this in an async handler so you don't have to use try catch.